File: DSA-with-C/array_c/delete.c
#include<stdio.h>
int main()
{
    int size, i, index;
    printf("Enter the size of the array : ");
    scanf("%d",&size);
    int array[size];
    for ( i = 0; i < size; i++)
    {
        printf("\nEnter the %d index no element of the array : ",i);
        scanf("%d",&array[i]);
    }

    printf("\nElements have entered : ");
    for ( i = 0; i < size; i++)
    {
        printf("%d ",array[i]);
    }
printf("\nEnter the index number of the element wnat to delete : ");
scanf("%d",&index);

for(i=index; i<size-1; i++)
{
    array[i]=array[i+1];
}
size--;
printf("\nElements after deletation : ");
for ( i = 0; i < size; i++)
    {
        printf("%d ",array[i]);
    }
printf("\n");
}

File: DSA-with-PYTHON/array_python/delete.py
from array import *
list=array('i',[])
size=int(input("Enter size of the element : "))
for i in range(size):
    list.append(int(input("Enter "+str(i)+" no element of the array : ")))
print("Elements before deletation : ", end=" ")
for i in range(len(list)):
    print(list[i], end=" ")
index=int(input("\nEnter the index of the element want to delete : "))
list.pop(index)
print("Elements after deletation : ", end=" ")
for i in range(len(list)):
    print(list[i], end=" ")
print(("\n"))
